src/osp/Active/SysPrefabInit.cpp

```diff
@@ -196,36 +197,77 @@ void SysPrefabInit::init_physics(
         auto const &rPrefabData = rResources.data_get<osp::Prefabs const>(
                 gc_importer, rPfBasic.m_importerRes);
 
-        auto const objects = rPrefabData.m_prefabs[rPfBasic.m_prefabId];
+        ArrayView<ActiveEnt const>  ents        = *itPfEnts;
+        lgrn::Span<int const>       objects     = rPrefabData.m_prefabs[rPfBasic.m_prefabId];
+        lgrn::Span<int const>       parents     = rPrefabData.m_prefabParents[rPfBasic.m_prefabId];
```

Comment:
these spans / views could be, themselves, const.

Reply:
Others comment that the amount of `const`s needed in C++ is getting silly. I did started using this style in other places; I think this is more readable:
```cpp
auto const ents     = ArrayView<ActiveEnt const>{*itPfEnts};
auto const objects  = lgrn::Span<int const>     {rPrefabData.m_prefabs[rPfBasic.m_prefabId]};
auto const parents  = lgrn::Span<int const>     {rPrefabData.m_prefabParents[rPfBasic.m_prefabId]};
```